### PR TITLE
location API JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
No QA required because only comments and a fix to a typo in a string logged to the program log.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6207/app_12143_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6207/HighFidelity-Beta-PR12143-309e7f7333dd524e8cc650697b185dcebb2bea0b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6207/HighFidelity-Beta-PR12143-309e7f7333dd524e8cc650697b185dcebb2bea0b.exe

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6213/app_12143_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6213/HighFidelity-Beta-PR12143-309e7f7333dd524e8cc650697b185dcebb2bea0b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6213/HighFidelity-Beta-PR12143-309e7f7333dd524e8cc650697b185dcebb2bea0b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6317/app_12143_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6317/HighFidelity-Beta-PR12143-02179e3efd3793c0824bf31d9c489fe0a03ee3cd.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6317/HighFidelity-Beta-PR12143-02179e3efd3793c0824bf31d9c489fe0a03ee3cd.dmg

File: libraries/networking/src/AddressManager.h [hunk 1]
```diff
@@ -247,7 +250,8 @@ public slots:
     void refreshPreviousLookup();
 
     /**jsdoc
-     * Save your current metaverse location in Interface's settings file.
+     * Update your current metaverse location in Interface's {@link Settings} file as your last-know address. This can be used
```
Comment (MiladNazeri) [inline libraries/networking/src/AddressManager.h]: typo on known

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6332/app_12143_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6332/HighFidelity-Beta-PR12143-f836a5f2b672b942857896680fb19b830be1fe92.exe

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6334/app_12143_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6334/HighFidelity-Beta-PR12143-f836a5f2b672b942857896680fb19b830be1fe92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6334/HighFidelity-Beta-PR12143-f836a5f2b672b942857896680fb19b830be1fe92.exe

**Pull request merged**